lnbits/extensions/satspay/crud.py

```diff
@@ -175,52 +175,5 @@ async def get_themes(user_id: str) -> List[SatsPayThemes]:
     return await get_config(row.user)
 
 
-################## SETTINGS ###################
-
-
-async def save_theme(data: SatsPayThemes, css_id: str = None):
-    # insert or update
-    if css_id:
-        await db.execute(
-            """
-            UPDATE satspay.themes SET custom_css = ?, title = ? WHERE css_id = ?
-            """,
-            (data.custom_css, data.title, css_id),
-        )
-    else:
-        css_id = urlsafe_short_hash()
-        await db.execute(
-            """
-            INSERT INTO satspay.themes (
-                css_id,
-                title,
-                "user",
-                custom_css
-                )
-            VALUES (?, ?, ?, ?)
-            """,
-            (
-                css_id,
-                data.title,
-                data.user,
-                data.custom_css,
-            ),
-        )
-    return await get_theme(css_id)
-
-
-async def get_theme(css_id: str) -> SatsPayThemes:
-    row = await db.fetchone("SELECT * FROM satspay.themes WHERE css_id = ?", (css_id,))
-    return SatsPayThemes.from_row(row) if row else None
-
-
-async def get_themes(user_id: str) -> List[SatsPayThemes]:
-    rows = await db.fetchall(
-        """SELECT * FROM satspay.themes WHERE "user" = ? ORDER BY "title" DESC """,
-        (user_id,),
-    )
-    return [SatsPayThemes.from_row(row) for row in rows]
```

Comment:
This is different to the earlier function, are we sure this function is not replacing the previous function and actually the one being used?

Reply:
You are right, it seems the other set of functions should be used - fixed in 2e572106232553a131931aa1ecde2511425715d6